model/sharedStimulus/parser/JsonQtiAttributeParser.php

```diff
@@ -77,12 +78,23 @@ private function hydrateXInclude(XInclude $xinclude, DOMDocument $document): XIn
      */
     private function addLanguageAttribute(DOMDocument $document, XInclude $xinclude): void
     {
-        if (isset($document->getElementsByTagName('div')->item(0)->attributes['lang']->nodeValue)) {
-            $xinclude->setAttribute(
-                'xml:lang',
-                $document->getElementsByTagName('div')->item(0)->attributes['lang']->nodeValue
+        $languageAttribute = $document->firstChild->getAttribute('xml:lang');
+
+        if (empty($languageAttribute)) {
```

Comment:
Would it be possible to get a string containing only whitespaces, or an invalid lang code as the value contained here (`<child lang=" ">`)?

```suggestion
        $languageAttribute = trim($document->firstChild->getAttribute('xml:lang'));

        if (empty($languageAttribute) || strlen($languageAttribute) < 2) {
```